src/usr/lib/rsetup/cli/u-boot-menu.sh

```diff
@@ -132,3 +132,48 @@ rebuild_overlays() {
     mv "$old_overlays" "${old_overlays}_old"
     mv "$new_overlays" "$old_overlays"
 }
+
+enable_overlays() {
+    __parameter_count_at_least_check 1 "$@"
+
+    local enable_overlays_list i dir_name file_name input_check=true
+
+    load_u-boot_setting
+
+    for i in "$@"
+    do
+        dir_name="$(dirname "$i")"
+        file_name="$(basename "${i%.disabled}")"
+
+        if [[ "$dir_name" == "." ]]
+        then
+            dir_name="$U_BOOT_FDT_OVERLAYS_DIR"
+        fi
+
+        if [[ "$(realpath "$dir_name")" != "$(realpath "$U_BOOT_FDT_OVERLAYS_DIR")" ]] ||
+           ! [[ -e "$dir_name/$file_name.disabled" ]] || ! [[ -e "$dir_name/$file_name.disabled" ]]
```

Comment:
```suggestion
        if [[ "$(realpath "$dir_name")" != "$(realpath "$U_BOOT_FDT_OVERLAYS_DIR")" ]]
```
后面的判断和下面的有些重复，可以在这个if链的最后加一个else，然后提示`$file_name: cannot find such overlay in '$U_BOOT_FDT_OVERLAYS_DIR'.`并设置flag